simpleSequential/test_stringToInteger.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from stringToInteger import stringToInteger
+
+class TestStringToInteger(unittest.TestCase):
+    def test_positiveStringInput(self):
```

Comment:
Let this be “test_positiveNumericalStringInput”

Reply:
Did you read this one? we don't have anything called "positive string".

Reply:
Acknowledge.

Reply:
I think you fix the wrong file. Did you check the file name?